app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -61,7 +61,19 @@ angular.module('Bastion.components').factory('Nutupane',
             params.paged = true;
             params.page = $location.search().page || 1;
             params['per_page'] = $location.search()['per_page'] || entriesPerPage;
-
+            /*
+             Start of changes: Fixes Issue #19960
+             Creating a flag "overrideAutoLoad" which will be part of params and will determine whether
+             the self loading of page needs to happen.
+             Use: Add an additional hash to the params named overrideAutoLoad and set it to true or false depending on your need.
+             If set to false, Nutupane.load (or refresh() etc..) needs to be called explicitly
+            */
```

Comment:
I think these start of and end of changes comments can be left out. Git history along with Redmine should provide enough history without cluttering the source code.